exporter/elasticsearchexporter/go.mod

```diff
@@ -26,6 +26,8 @@ require (
 	go.opentelemetry.io/otel/trace v1.27.0
 	go.uber.org/goleak v1.3.0
 	go.uber.org/zap v1.27.0
+	golang.org/x/exp v0.0.0-20240613232115-7f521ea00fb8
```

Comment:
I don't think we need x/exp? Go 1.21 has package slices

Reply:
nice catch, thanks!